src/write.jl

```diff
@@ -62,15 +62,16 @@ end
 function write_header(io::IO, file::File)
     h = file.header
     has_anno = file.annotations !== nothing
+    signal_count = h.n_signals + has_anno
     b = write_padded(io, h.version, 8) +
         write_padded(io, h.patient, 80) +
         write_padded(io, h.recording, 80) +
         _write(io, Dates.format(h.start, dateformat"dd\.mm\.yyHH\.MM\.SS")) +
-        write_padded(io, h.nb_header, 8) +
+        write_padded(io, 256 * (signal_count + 1), 8) +
```

Comment:
Where does the + 1 come from here?

Reply:
The header file itself is a constant 256 bytes.

Reply:
So it's 256 bytes of regular header stuff then 256 bytes in the header per signal?

Reply:
Yep!